stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -249,6 +249,49 @@ public struct Unsafe${Mutable}BufferPointer<Element>
 %end
   }
 
+  /// Accesses a contiguous subrange of the buffer's elements.
+  ///
+  /// The accessed slice uses the same indices for the same elements as the
+  /// original buffer uses. Always use the slice's `startIndex` property
+  /// instead of assuming that its indices start at a particular value.
+  ///
+  /// This example demonstrates getting a slice from a buffer of strings, finding
+  /// the index of one of the strings in the slice, and then using that index
+  /// in the original buffer.
+  ///
+%if Mutable:
+  ///     var streets = ["Adams", "Bryant", "Channing", "Douglas", "Evarts"]
+  ///     streets.withUnsafeMutableBufferPointer { buffer in
+  ///       let streetSlice = buffer[2..<buffer.endIndex]
```

Comment:
Thanks for adapting these docs! Just to keep everyone on their toes, we use four-space indentation in documentation examples, instead of the two used in code.

Reply:
oh? I hadn't noticed!